src/05_lists.py

```diff
@@ -8,22 +8,34 @@
 
 # Change x so that it is [1, 2, 3, 4]
 # YOUR CODE HERE
+x.append(4)
 print(x)
 
 # Using y, change x so that it is [1, 2, 3, 4, 8, 9, 10]
 # YOUR CODE HERE
+x.append(8)
```

Comment:
An easier/cleaner way to add x and y together is to extend rather than appending each one.